from chess.piece.exceptions import PieceValidationError


class KingValidationError(PieceValidationError):
    pass

